GameData/RealAntennas/PlanetPacks/JNSQ.cfg

```diff
@@ -0,0 +1,122 @@
+// RealAntennas for JNSQ
+//
+// author: Grimmas
+//
+
+@Kopernicus:FOR[JNSQ]:NEEDS[Kopernicus,JNSQ]
```

Comment:
Please change to BEFORE[JNSQ] or AFTER[JNSQ].
Doing FOR[JNSQ] is a bad practice, since it will declare the JNSQ pass as existing!  Thus making NEEDS[JNSQ] always true.

Reply:
Keeping in line with the other patch structures; RSS does this:
```
@Kopernicus:FOR[RealSolarSystem_Late]:NEEDS[Kopernicus,RealSolarSystem]
```

So might be best to use:
```
@Kopernicus:FOR[JNSQ_Late]:NEEDS[Kopernicus,JNSQ]
```

It should be fine to declare `JNSQ_Late` since... nothing should be looking for that.  If we want to be overly careful, perhaps `JNSQ_RealAntennas`